code/script.js

```diff
@@ -0,0 +1,100 @@
+// DOM SELECTORS
+const USER = 'Neaa99'
+const technigoRepos = document.getElementById('technigoRepos')
+const container = document.getElementById('container')
+const header = document.getElementById('header')
+
+const API_TOKEN = TOKEN || process.env.API_KEY;
```

Comment:
I know many people struggled with the API token, so well done for setting it up! I skipped it this time, but planning to come back to it when I have time.